app/src/main/java/com/eveningoutpost/dexdrip/ErrorsActivity.java

```diff
@@ -27,6 +27,7 @@
 
 /**
  * Created by Emma Black on 8/3/15.
+ * This is the old logs activity.
```

Comment:
What does "old logs" mean here? Is the class `ErrorsActivity` obsolete? If yes, we should mark it as deprecated and remove it in future versions.

Reply:
If you look at the ui of the error logs, and press the menu on the right there is an option there: "view old error log".
It is a similar screen, but looks (and implemented) differently. I'm not sure how many people are still using this.
Maybe it is time to remove it.